### PR TITLE
Explorer example shows what environment k8s runs your container in.

### PR DESCRIPTION
**Pull request by lavalamp**
Here's a little thing I made to help me debug my cluster while waiting for e2e to run. I think it's generally useful?

### CODE REVIEW

File: examples/explorer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2014 Google Inc. All rights reserved.
```
Comment (rjnagal) [inline examples/explorer/Dockerfile]: s/2014/2015/ ... also kubernetes authors.

File: examples/explorer/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Explorer is a little container for examining the runtime environment kubernetes produces for your pods.
+
+The intended use is to substitute gcr.io/google_containers/explorer for your intended container, and then visit it via the proxy. You can examine the environment variables to make sure kubernetes is doing what you expect, and you can examine the filesystem to make sure the mounted volumes and files are also what you expect. `pod.json` is supplied as an example.
+
+```
+$ alias kctl=../../../cluster/kubectl.sh
+$ kctl create -f pod.json
+$ kctl proxy &
+Starting to serve on localhost:8001
+$ curl localhost:8001/api/v1beta3/proxy/namespaces/default/pods/explorer:8080/vars/
```
Comment (rjnagal) [inline examples/explorer/README.md]: Add example for fs, dns, etc here too?

**Comment by rjnagal** [comment]
Just tried it, nice! :+1:

**Comment by lavalamp** [comment]
Comments fixed, image re-built & pushed.

**Comment by rjnagal** [comment]
LGTM, thanks @lavalamp

**Pull request merged**